src/sentry/filters/legacy_browsers.py

```diff
@@ -11,14 +14,53 @@
     'Safari': 6,
     'Edge': 0,
     'Opera': 15,
+    'Android': 4,
 }
 
 
+class LegacyBrowserFilterSerializer(serializers.Serializer):
+    # TODO: maybe find a serializer that handles multple values already
+    active = serializers.BooleanField()
+    subfilters = MultipleChoiceField(choices=[
+        'ie8',
+        'ie9',
+        'opera',
+        'android',
+        'safari'
+    ])
+
+
 class LegacyBrowsersFilter(Filter):
     id = 'legacy-browsers'
     name = 'Filter out known errors from legacy browsers'
     description = 'Older browsers often give less accurate information, and while they may report valid issues, the context to understand them is incorrect or missing.'
     default = False
+    serializer_cls = LegacyBrowserFilterSerializer
+
+    def is_enabled(self):
```

Comment:
`is_enabled()` implies that it should return a boolean, but this is returning the value of the `ProjectOption`, so we should coerce it to a boolean.